tools/chrome-debug-extension/rollup.config.js

```diff
@@ -0,0 +1,215 @@
+                values: replaceValues
+            })
+        ]
+    };
+
+    if (isProduction) {
+        browserRollupConfig.output.file = `browser/scripts/background.min.js`;
+        browserRollupConfig.plugins.push(
+            uglify({
+                ie8: true,
+                toplevel: true,
+                compress: {
+                    passes: 3,
+                    unsafe: true
+                },
+                output: {
+                    preamble: banner,
+                    webkit: true
+                }
+            })
+        );
+    }
+
+    return browserRollupConfig;
+};
+
+const generatePopup = (isProduction) => {
+    const browserRollupConfig = {
+        input: `dist-esm/popup.js`,
+        output: {
+            file: `browser/scripts/popup.js`,
+            banner: banner,
+            format: "umd",
+            name: "Microsoft.ApplicationInsights",
+            extend: true,
+            freeze: false,
+            sourcemap: true
+        },
+        plugins: [
+            nodeResolve({
+                browser: true,
+                preferBuiltins: true,
+                dedupe: ["react", "react-dom"]
+            }),
+            copy({
+                targets: [
+                    { src: "./images/*", dest: "browser/images" },
+                    { src: "./pages/*", dest: "browser/pages" },
+                    { src: "./styles/*", dest: "browser/styles" },
+                    { src: "./manifest.json", dest: "browser/" }
+                ]
+            }),
+            commonjs(),
+            replace({
+                preventAssignment: true,
+                delimiters: ["", ""],
+                values: replaceValues
+            })
+        ]
+    };
+
+    if (isProduction) {
+        browserRollupConfig.output.file = `browser/scripts/popup.min.js`;
+        browserRollupConfig.plugins.push(
+            uglify({
+                ie8: true,
+                toplevel: true,
+                compress: {
+                    passes: 3,
+                    unsafe: true
+                },
+                output: {
+                    preamble: banner,
+                    webkit: true
+                }
+            })
+        );
+    }
+
+    return browserRollupConfig;
+};
+
+const generateContentLoad = () => {
+    const browserRollupConfig = {
+        input: `dist-esm/contentLoad.js`,
+        output: {
+            file: `browser/scripts/contentLoad.min.js`,
+            banner: banner,
+            format: "iife",
+            extend: true,
+            freeze: false,
+            sourcemap: true
+        },
+        plugins: [
+            nodeResolve({
+                browser: true,
+                preferBuiltins: true
+            }),
+            commonjs(),
+            replace({
+                preventAssignment: true,
+                delimiters: ["", ""],
+                values: replaceValues
+            // }),
+            // uglify({
```

Comment:
We should re-enable these to minimize the amount of code loaded into the browser.